SU-HKKU/cardinal_service_profile
# CSD-358 Updated individual spotlight stories spacing

# READY FOR REVIEW

# Summary
- Updated individual spotlight stories to have top and bottom margin

# Needed By (Date)
- 08/25

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear cache
3. View individual spotlight story as an anonymous user. There should be spacing above and below the main spotlight content.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [CSD-358](https://stanfordits.atlassian.net/browse/CSD-358)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
